### ISSUE TITLE
[Question]: Unknown scheme for Frame.waitForEventInfo

### ISSUE THREAD

**Issue by tufbel** [microsoft/playwright-python#835]
### Your question

```
await page.fill("#username", username)
await page.fill("#password", password)
await page.click("input[name=\"submit\"]")
await page.wait_for_load_state("domcontentloaded")
```
An error occurred on the last line of the code:
```
[ERROR]: Future exception was never retrieved
future: <Future finished exception=Error('Unknown scheme for Frame.waitForEventInfo')> - [base_events.py:1707]
playwright._impl._api_types.Error: Unknown scheme for Frame.waitForEventInfo
```

**Comment by tufbel** [comment]
This error has never happened before.

**Comment by tufbel** [comment]
```browser = await playwright.chromium.launch(headless=False,slow_mo=1000)```
changed to:
```browser = await playwright.chromium.launch(headless=False)```

**Comment by mxschmitt** [comment]
Which version are you using?

**Comment by tufbel** [comment]
> Which version are you using? It should be fixed with 1.13.1

my version : 1.13.0
Thank you for your answer
